addons/docs/jest-transform-mdx.js

```diff
@@ -29,6 +29,9 @@ module.exports = {
       import { mdx } from '@mdx-js/react'
       ${mdx.sync(src, { compilers, filepath: filename })}
     `;
-    return getNextTransformer(filename, config).transformSource(filename, result, instrument);
+    
+    const jsFileName = `${filename}.js`;
```

Comment:
```suggestion
    const extension = path.extname(filename);
    const jsFileName = `${filename.slice(0, -extension.length)}.js`;
```

Needed to fix tests, see my comment in conversation above